source/chapter2-devicetree-basics.rst

```diff
@@ -1251,7 +1251,7 @@ interrupt controller.
       #address-cells = <1>;
       #size-cells = <1>;
 
-      open-pic {
+      open_pic: pic@13370000 {
```

Comment:
While you are here, the node name should really be "interrupt-controller" rather than "pic". The unit-address "@1337000" is not valid because there is no "reg" or "ranges" property. So you want:

open_pic: interrupt-controller {

And really you are adding a label, open_pic, rather than renaming the node name to that. Labels are a source level construct (except overlays make them more than that).

Reply:
Well, except this is all under a "simple-bus" node which is going to want unit-addresses along with "reg" properties... And the PCI node has similar issue... I guess compiling is an improvement, but a lot of warnings isn't great either.

I imagine this is not the only example with issues in the spec either.

Reply:
Thank you for the feedback - not 100% sure now what to do; I'll reiterate.

Reply:
so there is another example further up:

https://github.com/devicetree-org/devicetree-specification/blob/main/source/chapter2-devicetree-basics.rst?plain=1#L556-L560

for consistency, I suggest updating that as well. I'll just do this one in this PR for a start and see that it compiles without warnings.